docs/source/cli_help/config.rst

```diff
@@ -12,6 +12,7 @@ Note: ``config`` is not a command as such, just a help topic (there is no ``git
     When executing ``git machete github <subcommand>`` command, the following will happen:
 
```

Comment:
While you're here, I'd specify:

Line 3:  Documentation about available `git machete`` git config keys

Line 9: Git config keys: